pandunia-loge.csv

```diff
@@ -1217,85 +1219,84 @@ fili ja||||fan (-phile)||fan (-filo)||||||||||||||fani (-fiili)|miłośnik (fan,
 Filipinas|desha|PH||Philippines||Filipinas|||||||||||||Filipinoj|Filippiinit|Filipiny
 Filistin|desha|PS||Palestinian Territory||Palestina||||||||||||||Palestiina|Terytorium Palestyńskie
 filme|||eng:spa:fra:film, por:filme, deu:Film, rus:фильм (fil’m), tur:film, ara:فيلم (film), jpn:フィルム (firumu), may:pilem, swa:filamu|film (membrane, video)||película (film, membrana)||плева (фильм)||膜|フィルム (膜)||||||filamu|film|filmo|filmi|film
-filsofer||||philosopher||filósofo|||||||||||mwanafalsafa||filozofo|filosofi|filozof
-filsofi||||philosophic||filosófico|||||||||||||filozofia|filosofinen|filozoficzny
-filsofia||||philosophy|philosophie|filosofía|filosofia|философия|فَلْسَفَة|哲学|哲学|철학|triết học|दर्शन (फ़लसफ़ा)||filsafat|falsafa|felsefe|filozofio|filosofia|filozofia
-filter|||eng:filter, spa:por:filtro, fra:tur:filtre, rus:фильтр (filʹtr), hin:फ़िल्टर (filṭar), ben:ফিল্টার (philṭar), jpn:フィルター (firutā)|filter||filtro|||||||||||||filtri|suodattaa|filtr
+fili sofi ja||||philosopher||filósofo|||||||||||mwanafalsafa||filozofo|filosofi|filozof
+fili sofi di||||philosophic||filosófico|||||||||||||filozofia|filosofinen|filozoficzny
+fili sofi||||philosophy|philosophie|filosofía|filosofia|философия|فَلْسَفَة|哲学|哲学|철학|triết học|दर्शन (फ़लसफ़ा)||filsafat|falsafa|felsefe|filozofio|filosofia|filozofia
```

Comment:
I think this should be an independent root, **filsofi**, because the original Greek word order is the opposite ("love of wisdom") and the Pandunia interpretation, "love wisdom", sounds like something about romantic relationships.

Reply:
that makes sense.  then why **filsofi** instead of **filosofi**?

Reply:
**filsofi** is shorter. Three syllables is my unofficial limit for root words. The Arabic-influenced half of the etymology does without -o-.
eng:philosophy, fra:philosophie, spa:por:filosofia, rus:философия (filosofiya), ara: فَلْسَفَة (falsafa), hin:फ़लसफ़ा (falsafā), may:filsafat, swa:falsafa, tur:felsefe